core/modules/random_drop.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BESS_MODULES_RANDOM_DROP_H_
+#define BESS_MODULES_RANDOM_DROP_H_
+
+#include "../module.h"
+#include "../module_msg.pb.h"
+#include "../utils/random.h"
+
+const uint kRange_ = 1000000;  // for granularity
```

Comment:
Sorry, just one more small change -- it would be best for this to be:

static const uint32_t kRange = 1000000;

And be defined in the public section of the RandomDrop class.